crates/node/src/chain_spec.rs

```diff
@@ -24,7 +24,7 @@ pub type ChainSpec = sc_service::GenericChainSpec<GenesisConfig>;
 pub type DevChainSpec = sc_service::GenericChainSpec<DevGenesisExt>;
 
 /// Starknet testnet SN_GOERLI
-pub const CHAIN_ID_STARKNET_TESTNET: u128 = 0x534e5f474f45524c49;
+pub const _CHAIN_ID_STARKNET_TESTNET: u128 = 0x534e5f474f45524c49;
```

Comment:
It is not used anymore?

Reply:
no it was used to build felt252wrapper of chain_id used here https://github.com/keep-starknet-strange/madara/pull/867/files#diff-6ee0110ea69cf8292e5a04f1098288a3c0f2c92a765a31fdbfeb70499ae78264L224

will remove it.